docs/REST API/rest-api.md

```diff
@@ -7373,22 +7373,20 @@ This allows you to easily configure scheduled queries that will impact a whole t
 
 - [Run script](#run-script)
 - [Get script result](#get-script-result)
-- [Run live script](#run-live-script)
```

Comment:
Let's not delete any of these endpoints from the docs (we can rename for clarity instead)